### DISCUSSION TITLE
Shorter Mesh Sequence for Screw Adjustments

### DISCUSSION THREAD

**Discussion by Japander** [Jyers/Marlin#545] [Q&A] [answered]
Hey everyone,

i`m with the 15x15 UBL Mesh Firmware, and just changed the springs for my bed.

I want to create a less acurate mesh to see with Octoprint how much i have to adjust the screws, so the bed is mostly even, and create the "real mesh" afterwards.

Can anyone help me? I searched the menus but didnt found the right thing... Is this possible?

Best regards from Hamburg, Germany
Marko

**Comment by smooreace** [comment]
Install the appropriate firmware for the mesh 'frequency' you want.

**Comment by Jyers** [comment, marked as answer]
There is a feature which essentially does what your describing called assisted tramming. Implementing this into the firmware is currently underway.
